ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/ScenarioLoader.scala

```diff
@@ -150,36 +149,42 @@ object ScenarioLoader {
   }
 
   private def getCandidateScenarios(
-      packages: DamlPackageContainer,
+      packages: InMemoryPackageStore,
       scenarioQualName: Ref.QualifiedName
   ): List[(Ref.Identifier, Definition)] = {
-    packages.packages.flatMap {
-      case (packageId, pkg) =>
-        pkg.lookupIdentifier(scenarioQualName) match {
-          case Right(x) => List((Ref.Identifier(packageId, scenarioQualName), x))
-          case Left(_) => List()
-        }
-    }(breakOut)
+    packages
+      .listLfPackagesSync()
+      .flatMap {
+        case (packageId, _) =>
+          val pkg = packages.getLfPackageSync(packageId).get
```

Comment:
Do not use `Option#get`.

```suggestion
          val pkg = packages.getLfPackageSync(packageId) getOrElse sys.error(s"I am actually explaining the difficulty with $packageId")
```